src/main.cpp

```diff
@@ -580,7 +580,7 @@ int main ( int argc, char** argv )
 #ifdef HEADLESS
     QCoreApplication* pApp = new QCoreApplication ( argc, argv );
 #else
-# if defined (Q_OS_IOS)
+# if defined (  Q_OS_IOS  )
```

Comment:
Have you now used two spaces on each side of the bracket? :-) So you inserted a total of 4 spaces but only 2 are needed ;-)

Reply:
Ok. Is this ok now?

Reply:
Yes, thanks.